modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/resolver/AttributePublisherImpl.java

```diff
@@ -44,40 +40,17 @@ public void publish(String name, String nameFormat, String... values) {
 
 		List<XMLObject> attributeXmlObjects = attribute.getAttributeValues();
 
-		AttributeValue[] attributeValues = TransformUtil.transform(
-			values,
-			value -> new AttributeValueWrapper(
-				OpenSamlUtil.buildAttributeValue(value)),
-			AttributeValue.class);
+		List<XMLObject> xmlObjects = new ArrayList<>();
 
-		Stream<AttributeValue> stream = Arrays.stream(attributeValues);
+		for (String value : values) {
```

Comment:
@jeffwu0724 We can also use `TransformUtil` here.

Reply:
changed to use transformUtil here